org/org.yaml

```diff
@@ -1,7 +1,7 @@
 orgs:
   tektoncd:
     admins:
-    - priyawadhwa
```

Comment:
I think you need to move yourself to the `members:` list below, otherwise peribolos will try to remove you from the org completely (and fail).

Reply:
thanks! fixed :)

Reply:
I think the reverse also needs to happen (move me out of members) 🙃

Reply:
haha thanks! done